src/types/redis_bloom_chain.cc

```diff
@@ -151,6 +151,7 @@ rocksdb::Status BloomChain::Add(const Slice &user_key, const Slice &item, int *r
     s = bloomCheck(bf_key_list[check_index], item_string, ret);
```

Comment:
I think we should add another variable to identify the check result(e.g. exists or check_ret) instead of resuing the add result.